packages/components/src/internal/components/samples/SampleLineageGraph.tsx

```diff
@@ -108,24 +118,28 @@ export class SampleLineageGraph extends PureComponent<Props, State> {
     }
 
     render() {
-        const { sampleLsid, goToLineageGrid, onLineageNodeDblClick, groupTitles } = this.props;
+        const { sampleLsid, sampleID, goToLineageGrid, onLineageNodeDblClick, groupTitles, groupingOptions } = this.props;
+
+        let grouping = { ...(groupingOptions ?? {childDepth: DEFAULT_LINEAGE_DISTANCE}), generations: LINEAGE_GROUPING_GENERATIONS.Specific};
         return (
             <Panel>
                 <Panel.Body>
                     <Button bsStyle="success" onClick={goToLineageGrid}>
                         Go to Lineage Grid
                     </Button>
                     {this.renderFilter()}
+
                     <LineageGraph
                         lsid={sampleLsid}
-                        grouping={{ generations: LINEAGE_GROUPING_GENERATIONS.Specific }}
+                        grouping={grouping}
                         filters={this.getLineageFilters()}
                         navigate={onLineageNodeDblClick}
                         groupTitles={groupTitles}
                         runProtocolLsid={this.getRunProtocolLsid()}
                     />
+                    <LineageDepthLimitMessage className={"lineage-graph-generation-limit-msg"} maxDistance={grouping.childDepth} nodeName={sampleID}/>
```

Comment:
I think this was mentioned by Nick, but this className looks to match the default value of `LineageDepthLimitMessage`

Reply:
Removed